src/common/schedulers/slurm_commands.py

```diff
@@ -194,7 +194,7 @@ def set_nodes_drain(nodes, reason):
 
 def set_nodes_power_down(nodes, reason=None):
```

Comment:
why don't we keep a single function `set_nodes_power_down` and remove `set_nodes_down_and_power_save` since they are both doing the same? We might want to add the retry annotation on top of `set_nodes_power_down`

Reply:
Removed `set_nodes_down_and_power_save`